dredd/api-description.yml

```
@@ -1215,11 +1215,15 @@ definitions:
         type: string
         description: Episode plot
       content:
-        type: array
+        type: object
         description: Additional content
-        items:
-          type: string
-          enum: [NFO, thumbnail]
+        properties:
+          hasNfo:
+            type: boolean
+            description: Whether the episode has an .nfo file
+          hasNfo:
```

```suggestion
          hasTbn:
```
---
But personally, I think the keys should be named `nfo` and `thumbnail`.
```yml
      content:
        type: object
        description: Additional content
        properties:
          nfo:
            type: boolean
            description: Whether the episode has an .nfo file
          thumbnail:
            type: boolean
            description: Wheter the episode has a thumbnail file
```